provider/resources.go

```diff
@@ -2094,9 +2094,10 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 					Source: "vpc_peering_options.html.markdown",
 				},
 			},
-			"aws_default_vpc":  {Tok: awsResource(ec2Mod, "DefaultVpc")},
-			"aws_vpc":          {Tok: awsResource(ec2Mod, vpcMod)},
-			"aws_vpc_endpoint": {Tok: awsResource(ec2Mod, "VpcEndpoint")},
+			"aws_default_vpc":            {Tok: awsResource(ec2Mod, "DefaultVpc")},
+			"aws_vpc":                    {Tok: awsResource(ec2Mod, vpcMod)},
+			"aws_vpc_encryption_control": {Tok: awsResource(ec2Mod, "EncryptionControl")},
```

Comment:
```suggestion
			"aws_vpc_encryption_control": {Tok: awsResource(ec2Mod, "VpcEncryptionControl")},
```

Since we are putting it in the ec2 mod it should probably have Vpc as the prefix to match what we do for the other resources.